### DISCUSSION TITLE
Question About Blank Screens

### DISCUSSION THREAD

**Discussion by mifjpn** [dakanji/RefindPlus#28] [General]
I don't get a blank screen.
(i7 8700,ASUS Prime H370-A,RX-570)

And I looked at the log of how at least two blank screens happened.

What is different in the log
”1: 735 0:010 Scan Volumes ...”
There is no UUID displayed inside.

And they also specified by UUID, healed from the blank screen and couldn't boot when the entry came out.

Is it possible that the UUID is not scanned due to some environmental difference?

I have also been informed of the problem.

If you have time, I would like to know it.
Also, if such a UUID is not scanned, is the volume specification on the GTP label valid?

[21f08z0541.log](https://github.com/dakanji/RefindPlus/files/6102332/21f08z0541.log)

**Comment by dakanji** [comment]
I'm afraid I don't understand your question. You need to be a bit clearer.

**Reply by mifjpn** [reply]
Thank you for picking it up.
Thank you for making a great app.

I can only get information in pieces.
I would like to ask the person who has used this wonderful app because the update caused it to go wrong and he is supposed to press the F8 button again.

As a general rule, when comparing, it is the principle of debugging that they must be the same except for making a difference.

I'm sorry I can only suggest because it is far from the principle.

However, the log of the person I heard that it does not work is as follows.
[21d05v5653.log](https://github.com/dakanji/RefindPlus/files/6102680/21d05v5653.log)
[21d05v0204.log](https://github.com/dakanji/RefindPlus/files/6102681/21d05v0204.log)

On the other hand, my log that has been successful is as follows.
[21f08z0541.log](https://github.com/dakanji/RefindPlus/files/6102684/21f08z0541.log)


Is it possible that the UUID is not scanned due to some environmental difference?

If you have time, I would like to know it.
Also, if such a UUID is not scanned, is the volume specification on the GTP label valid?

Thank you.

**Comment by dakanji** [comment]
As you said, things should be the same to be able to determine differences but you have this:
![Screen Shot 2021-03-08 at 21 26 08](https://user-images.githubusercontent.com/1128876/110364564-fb7fd880-8054-11eb-8805-a90ee13d9d45.jpg)

while the other has this:
![Screen Shot 2021-03-08 at 21 28 08](https://user-images.githubusercontent.com/1128876/110364740-37b33900-8055-11eb-8ab0-bef7350f25d0.jpg)

and this:
![Screen Shot 2021-03-08 at 21 29 39](https://user-images.githubusercontent.com/1128876/110364918-6df0b880-8055-11eb-8973-09a17f8d29aa.jpg)

Aligning the versions is the first thing to do and if there is still a different outcome after this, then there is something to look at. 

Without this, there are too many potential variables including the basic one of the possibility that something in those older versions that have now been addressed in the newer one.

**Reply by mifjpn** [reply]
I apologize for doing something I'm sorry for you.

I introduced it to many people because I thought it had a wider range of support than refind (not plus). (Of course, according to the meaning of the word "plus")

I also thought that the update here was quick, fair, and You were taking on various challenges.

In a Japanese forum, a friend told me, "RefindPlus has been improved on the assumption that it will be used in combination with OpenCore on Mac Pro (especially Clover is not recommended at all).
If you want to do it with an AT compatible machine, it is better to use the original refind. "

I thought it will have  ”universal improvement and fair”, but I mistake to think it.

Also, if you don't want to be "universal, improved and fairer", I apologize for taking your valuable development time and  I think I have to cancel introduce refind on my page.

I'm really sorry.

Thank you.

If the English translation is bad and misleading, I apologize in advance.

**Reply by dakanji** [reply]
Thanks for explaining your views.

> I apologize for doing something.

Its ok. there is nothing to apologise for. 

You simply asked a question which I tried to answer. This is the purpose of this place.

> I introduced it to many people because I thought it had a wider range of support than refind (not plus). 

"Plus" means it is rEFInd plus additions.

> I also thought that the update here was quick, fair, and You were taking on various challenges.

Not sure about "fair" (maybe needs a different translation) but yes, there is an attempt to update regularly and the various additions does should show that taking on various challenges is happening.  

> In a Japanese forum, a friend told me, "RefindPlus has been improved on the assumption that it will be used in combination with OpenCore on Mac Pro (especially Clover is not recommended at all).
If you want to do it with an AT compatible machine, it is better to use the original refind."

> I thought it will have ”universal improvement and fair”, but I mistake to think it.

> Also, if you don't want to be "universal, improved and fairer", I apologize for taking your valuable development time and I think I have to cancel introduce refind on my page.

Very sorry, but struggling with what you mean by "fair".  What I can say is that while RefindPlus does tilt towards OpenCore on a MacPro, as this is what I personally use, it is not specifically designed for one thing or another, and does not recommend anything. It just does its job which is to allow users to boot stuff.

I use OpenCore and can test this. Accordingly, some fine details are towards OpenCore.  However, by coincidence, I just added something specifically Clover related a couple of days ago: https://github.com/dakanji/RefindPlus/commit/c06ddc963f01b6c9b131ceae35cdc5972c51864e

Anyway, you and your friends need to decide what you think will work best for you. The original rEFInd has been around for a very long time and has a great track record. It has started receiving updates again (some of them flowing back from RefindPlus). 

If you and your friends feel this is best to work with that version, nothing bad can  happen with this decision. 

All the best!

**Comment by vhalkola** [comment]
This probably belongs here. Both Refind and Refind Plus stall at a point where only the banner is shown, otherwise just blank light grey screen. I also have a Mac Pro 3,1 on which things work as they should, so this might have something to do with HP's UEFI. On my old HP computer (HP Compaq Elite 8200, from which I moved GPU to my new computer) there were also no problems; the problem appeared when I switched to HP Z440. As the log shows, the process stalls after scanning for bootable kernels. I'm yet to test whether older versions of RefindPlus exhibit the same behaviour.

[21f24p2433.log](https://github.com/dakanji/RefindPlus/files/6196832/21f24p2433.log)

DA_EDIT: Issue https://github.com/dakanji/RefindPlus/issues/40

**Reply by dakanji** [reply]
Thanks for the report.

Please create an issue about this and follow the issue template.
